app/assets/javascripts/index/new_note.js

```diff
@@ -77,23 +77,20 @@ OSM.NewNote = function (map) {
     marker.addTo(noteLayer);
   }
 
-  function newHalo(loc, a) {
-    var hasHalo = halo && map.hasLayer(halo);
-
-    if (a === "dragstart" && hasHalo) {
-      map.removeLayer(halo);
-    } else {
-      if (hasHalo) map.removeLayer(halo);
-
-      halo = L.circleMarker(loc, {
-        weight: 2.5,
-        radius: 20,
-        fillOpacity: 0.5,
-        color: "#FF6200"
-      });
+  function addHalo(latlng) {
+    if (halo) map.removeLayer(halo);
+    halo = L.circleMarker(latlng, {
+      weight: 2.5,
+      radius: 20,
+      fillOpacity: 0.5,
+      color: "#FF6200"
+    });
+    map.addLayer(halo);
```

Comment:
The vertical whitespace was there to make this more readable...

Reply:
restored whitespace